README.md

```diff
@@ -23,10 +23,21 @@ specification to describe this ability as well, and move the document to the sta
   * todo
 
 ## Goals
-  * todo
+* Enable a web app to register tasks to be run periodically, with a retry logic baked in, which has an exponential backoff.
+* Provide the web app a way to query the list of previously registered tasks.
+* Enable the web app to unregister previously registered tasks.
+* Supported platforms: Android, Windows, Linux, ChromeOS, Mac. 
 
 ## Non-goals
-  * todo
+* Triggering events at a specific time is an explicit non-goal. The alarms API enables that.
```

Comment:
"_What alarms API?_"

Reply:
Updated to Scheduled Task API and added a link.